lib/graphql/account.ts

```diff
@@ -37,6 +37,44 @@ const GET_ACCOUNT_STATS = `query {
   }
 }`;
 
+const GET_SOLARIS_BALANCE = `query {
+  viewer {
+    mainAccount {
+      solarisBalance {
+          balance {
+            value,
```

Comment:
The commas in the query can be removed to make everything consistent, even though graphql works with commas but it is not recommended